api-spec/filters.md

```diff
@@ -0,0 +1,133 @@
+STAC Core is intending to provide a minimal set of features so that there is a low bar to entry for catalog
+implementations. The core concepts in STAC are Spatial and Temporal data, so those are the only filters we want
+to support in Core.
+
+Filters should be as simple as possible for a client to construct. Since we are returning json, we want to also
+consume json. This allows clients to support filtering without additional tools. So the core filters will be 
+JSON based. The search enpoint will accept application/json format queries, and GET on the collection will 
+support URL encoded JSON. POST search the is recommended way to filter results to avoid the URL encoding 
+issues that can happen with GET.
+
+Searching using POST will accept a JSON object where the top level keys are specifying which type of filter
+to apply. Those same top level key names can be used as query string parameters for the GET request. Items in the collection 
+should match all filters to be returned when querying. This implies a logical AND operation. If
+an OR operation is needed, it should be specified through an extension filter.
+
+For the spatial query, we are starting with the simplest form for a client to generate: a bbox. The bbox query 
+is a simple array of WGS84 Longitude and Latitude values. They are in this order: [west, south, east, north]. 
+This query will perform an intersects operation on the geometry values of the items in the catalog. Some GeoJSON 
+objects may provide a bbox property in addition to geometry, but it should not be used for the bbox filter since
+it is an optional field in GeoJSON.
+
+example
+```
+{
+  "bbox": [-180,-90,180,90]
+}
+```
+
+The temporal query will be based on ISO 8601 and should support time ranges as well as equality. To support range
```

Comment:
Do we want to switch this to RFC 3339 for consistency? I forget the subtleties of the difference between the two, like if RFC 3339 is sufficient? (should also link to whichevery one we use)

Reply:
Updated, RFC 3339 is more explicit and is probably the better one to use.